### DISCUSSION TITLE
Is "trust browser for N days" functionality possible?

### DISCUSSION THREAD

**Discussion by truthdoug** [django-otp/django-otp#153] [Q&A]
Does this extension allow for a site to only ask for TOTP code every `N` days rather than on every login?

My org's security policy dictates short admin sessions but allows for MFA checks that are less frequent.

If this is not natively supported, I'd appreciate a hint about how where we might add this if we were to fork this.

**Comment by psagers** [comment]
[django-otp-agents](https://django-otp-agents.readthedocs.io/) may be what you’re looking for.

**Reply by truthdoug** [reply]
Thanks, I'll look into this!
